ui/src/components/EntitySearch/FacetedEntitySearch.jsx

```diff
@@ -203,15 +262,30 @@ export class FacetedEntitySearch extends React.Component {
   }
 }
 const mapStateToProps = (state, ownProps) => {
-  const { query, result } = ownProps;
+  const { query } = ownProps;
+  const searchConfig = getSearchConfig();
+  const facets = searchConfig?.facets || defaultFacets.map(getGroupField);
+  const columns = searchConfig?.columns || defaultColumns.map(getGroupField);
```

Comment:
I've not seen this before, what do these two line denote? is searchConfig?.facets a variable, or a shorthand if statement, or something specific to react that I'm just not familiar with?

Reply:
Ahh yes :) fun new ES6 optional chaining syntax - which allows for a more compact way to catch attempts to access props on undefined objects

Reply:
Nice. I'm obviously losing touch with the state of ES. I'll have a look into this. Thanks for the heads up.

Reply:
I think it's pretty much impossible to keep track of this crazy ever-changing javascript monster XD